src/pipelinedefinitionstatus.hpp

```diff
@@ -29,6 +29,7 @@
 #include "tensorflow_serving/apis/prediction_service.grpc.pb.h"
 #pragma GCC diagnostic pop
 
+#include "modelversionstatus.hpp"
```

Comment:
I think this is the moment we should separate dependence between ModelVersionState & ModelVersionStatus.
Separate headers.